lib/avromatic/model/attributes.rb

```diff
@@ -51,6 +51,10 @@ def schema_fields_differ?(name)
         end
 
         def define_avro_attributes(schema)
+          if schema.type_sym != :record
```

Comment:
I wrote it that way initially then decided to avoid the `unless` instead. Lesser of two evils ...?